### DISCUSSION TITLE
AppEngine --generated_dir

### DISCUSSION THREAD

**Discussion by haninaguib** [GoogleCloudPlatform/cloud-code-intellij#3076] [Q&A]
How can I specify the directory where generated files are created for a local dev-appserver?
In the command line you can use the --generated_dir flag, but I don't see an option in the plugin. It currently defaults to target/{app}/WEB-INF/classes/appengine-generated. This is a problem as a mvn clean will wipe it.

**Comment by etanshaul** [comment]
Hi @haninaguib . This is currently a limitation in the feature. We will consider this a feature request. I'll update if I can find a workaround in the meantime.
